src/Symfony/Component/JsonPath/JsonCrawler.php

```diff
@@ -301,7 +326,7 @@ private function evaluateFilterExpression(string $expr, array $context): bool
         }
 
         // function calls
-        if (preg_match('/^(\w+)\((.*)\)$/', $expr, $matches)) {
+        if (preg_match('/^(\w+)\s*\(\s*(.*)\s*\)$/', $expr, $matches)) {
```

Comment:
```suggestion
        if (preg_match('/^(\w++)\s*+\((.*)\)$/', $expr, $matches)) {
```

And then trimming `matches[1]` (as the current regex does not even guarantee that the trailing spaces will be matched by the `\s*` part so you still need trimming)